rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/locations/Locations.java

```diff
@@ -41,6 +41,29 @@
 import io.usethesource.vallang.IValue;
 
 public class Locations {
+    public static ISourceLocation toClientLocation(ISourceLocation loc) throws IOException {
```

Comment:
I think we should add some notes on these functions to help future developer pick the right one, as to remain compatible with this redirection feature.